agent/ebs/watcher.go

```diff
@@ -198,3 +307,27 @@ func (w *EBSWatcher) handleEBSAckTimeout(volumeId string) {
 		w.removeEBSAttachment(volumeId)
 	}
 }
+
+func (w *EBSWatcher) sendEBSStateChange(ebsvol *apiebs.ResourceAttachment) error {
+	if ebsvol == nil {
+		return errors.New("ebs watcher send EBS state change: nil volume")
+	}
+	go w.emitEBSAttachedEvent(ebsvol)
```

Comment:
We want to call this method asynchronously because `w.taskEngine.StateChangeEvents() <- attachmentChange` might block? Not blocking this PR but have we ever considered making `taskEngine.stateChangeEvents` buffered?